projects/media-viewer/src/lib/toolbar/main-toolbar/sub-toolbar/sub-toolbar.component.html

```diff
@@ -16,6 +16,11 @@
               data-l10n-id="page_rotate_cw" (click)="rotateCw()">
         <span data-l10n-id="page_rotate_cw_label">Rotate Clockwise</span>
       </button>
+      <button *ngIf="toolbarButtons.showRotate" class="secondaryToolbarButton dragScroll" title="Drag Scroll" tabindex="6"
```

Comment:
why does this use showRotate?

Reply:
at the time i didnt create a toolbar event for the drag